docs/roadmap.md

```diff
@@ -7,23 +7,30 @@ If you are interested in helping to define and/or contribute to this roadmap, pl
 # Completed Releases
```

Comment:
Doesn't need to be changed, but what are your thoughts on moving the "completed releases" section to the end, so that the "upcoming (planned) releases" appears closer to the top of the document, since this just rehashes the information from the CHANGELOG doc?

Or do we need a section for completed releases, since the roadmap is a forward looking document?

Reply:
The "Completed Releases" header could just become a link to the CHANGELOG (and the list of completed releases removed).

Reply:
Done